crates/sui-core/src/unit_tests/transaction_manager_tests.rs

```diff
@@ -235,12 +250,35 @@ async fn transaction_manager_read_lock() {
         )
         .unwrap();
 
+    // Enqueue one transaction with two readable shared objects, one is above shared object.
+    let shared_version_2 = 2000.into();
+    let shared_object_arg_read_2 = ObjectArg::SharedObject {
+        id: shared_object_2.id(),
+        initial_shared_version: 0.into(),
+        mutable: false,
+    };
+    let transaction_read_2 = make_transaction(
+        gas_objects[3].clone(),
+        vec![CallArg::Object(shared_object_arg_read_2)],
```

Comment:
This is only one shared object input?

Reply:
Good catch! Revised. I guess the test workflow is the same